### ISSUE TITLE
Can't install appimagers or single executables  on linux

### ISSUE THREAD

**Issue by jpenguin** [itchio/itch.io#1334]
Several linux games are distributed as *.appimage files and sometime executables as *.x86/*.x86_64  The itch client thinks it needs to process these files but fails.  All it needs to do is 'chmod +x' these files

https://dale.itch.ovh/8sXb4X2u7OPPWGaPN5iGAuGTEtCi0zzwo8RPgEkK6JLxL4NaJn13d9IfsOBhAWHK

**Comment by lunarcloud** [comment]
Yeah, I'm trying to play a game whose executable is just the game's name and the itch app won't install.

[itch.txt](https://github.com/itchio/itch.io/files/10147446/itch.txt)
